src/main/java/org/littleshoot/proxy/ServerConnectionFailureHttpResponseComposer.java

```diff
@@ -0,0 +1,9 @@
+package org.littleshoot.proxy;
+
+
+import io.netty.handler.codec.http.FullHttpResponse;
+import io.netty.handler.codec.http.HttpRequest;
+
+public interface ServerConnectionFailureHttpResponseComposer {
```

Comment:
well, it should contain `failure` in it as it expects throwable cause.. it can not be used for constructing OK response.. so `FailureHttpResponseComposer`? @mjallday